corehq/ex-submodules/couchforms/_design/filters/xforms.js

```diff
@@ -11,10 +11,12 @@ function(doc, req)
     switch (doc_type) {
         case "XFormInstance":
         case "XFormArchived":
+        case "XFormError":
         case "XFormDeprecated":
         case "XFormDuplicate":
         case "XFormInstance-Deleted":
         case "HQSubmission":
+        case "SubmissionErrorLog":
             return true;
         default:
             return false;
```

Comment:
This is going to be a big reindex. Another option could be to create a new couchapp with just this filter